### PR TITLE
[DOC] Adding instructions related to parameter settings in ON YARN scenarios

### PR DESCRIPTION
**Pull request by BsoBird**
Adding instructions related to parameter settings in ON YARN scenarios

### Purpose

null

For paimon-action, if the user uses FLINK ON YARN, the current documentation does not describe how to set the parameters in this scenario. Given the large number of users using FLINK ON YARN, I think we need to add documentation.

### CODE REVIEW

File: docs/content/maintenance/dedicated-compaction.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Example: compact table
     --catalog-conf s3.secret-key=*****
 ```
 
-You can use `-D execution.runtime-mode=batch` to control batch or streaming mode. If you submit a batch job, all
+You can use `-D execution.runtime-mode=batch` or `-yD execution.runtime-mode=batch` to control batch or streaming mode(The -yD option is used for the ON-YARN scenario). If you submit a batch job, all
```
Comment (yuzelin) [inline docs/content/maintenance/dedicated-compaction.md:115]:
```suggestion
You can use `-D execution.runtime-mode=batch` or `-yD execution.runtime-mode=batch` (for the ON-YARN scenario) to control batch or streaming mode. If you submit a batch job, all
```

Comment (BsoBird) [reply docs/content/maintenance/dedicated-compaction.md:115]: Tks

Comment (BsoBird) [reply docs/content/maintenance/dedicated-compaction.md]: @yuzelin

**Comment by yuzelin** [comment]
@BsoBird Thanks for contributing.

**Pull request merged**